AI/AIInterface.h

```diff
@@ -264,7 +264,7 @@ namespace AIInterface {
 
     /** @brief Return the canonical AI directory path
      *
-     * The value depends on the ::OptionsDB `resource-dir` and `ai-path` keys.
+     * The value depends on the ::OptionsDB `content.path` and `ai-path` keys.
```

Comment:
not a fan of this change... "content" and "resource" are not the same thing.

Reply:
Well, the AI aspect of it, where you've noted your comment here, is just harmonizing the AI interface comments with the changes this PR makes elsewhere, which on its own seems appropriate to me.

If your main concern is about the underlying change of the option name from 'resource-dir' to 'content.path', well, @dbenage-cx might have his own arguments to make, but I'll go ahead and opine that the change seemed reasonable to me-- the word 'resource' is indeed a bit broader than the word 'content', but the more specific word 'content' seems reasonable to me in this case and I think the significance of a 'content.path' option might easily be more clear to new users than for a 'resource-dir' option.  Is there some notable way that you are thinking 'content.path' would be inaccurate/misleading, or is there some other concern?

Reply:
The reason for picking `content` was due to references using the term "game content" as well as aligning with "content scripting" (focs). I'm happy to change the group to `resource.*` if that is the preference.

In either case, I need to update README files (was not searching *.md).

Reply:
Pushing this change without squashing, so that it may be easily dropped if decided.